common/tensorrt_common/include/tensorrt_common/tensorrt_common.hpp

```diff
@@ -184,12 +184,19 @@ class TrtCommon  // NOLINT
    */
   void setup();
 
+#if (NV_TENSORRT_MAJOR * 10000) + (NV_TENSORRT_MINOR * 100) + NV_TENSOR_PATCH >= 80500
```

Comment:
Hey @amadeuszsz, thank for your feedback.

I've updated the multiplier as you suggested and wrapped the old API functions with macros as well.

related commit: https://github.com/autowarefoundation/autoware.universe/pull/9018/commits/a64285b993b713ed29922490b90995c936773cc9

---

> Note:
> We work on upgrading all packages to use new API via tensorrt_common to keep backward compatibility. It requires general approach which will meet all perception packages requirements, so please be aware of some extra changes in tensorrt_common from our side. I know you are working on RTMDet feature and we might ask you for tiny RTMDet refactor in case of next tensorrt_common API change.

OK.

Reply:
1. Sorry for misunderstanding, I was not clear. My idea was not to add else condition, but enable enqueueV3 for TRT > 8.5 and enable enqueueV2 for TRT < 10.0.
2. I guess it should be 8500, not 8050, right?

Reply:
sorry my bad, it should be okay now. Can you check it again?

related commit: https://github.com/autowarefoundation/autoware.universe/pull/9018/commits/a40bcc98dee494654230f9b7898258971adfe4e0